lib/ReactViews/GyroscopeGuidance/GyroscopeGuidance.jsx

```diff
@@ -27,6 +30,10 @@ const CompassWrapper = styled(Box).attrs({
   width: 64px;
   height: 64px;
   margin-right: 10px;
```

Comment:
Also maybe this could be turned into a `<Spacing right />` component?

Reply:
Good pickup - done